sql/_05_plcsql/_01_testspec/_02_declaration/_02_variable/_02_uninitialized/cases/02_02_02-03_normal_function.sql

```diff
@@ -37,10 +37,8 @@ call t();
 
 
 create or replace procedure t  as
-    a varchar := USER();
+    a varchar;
```

Comment:
I think it would be better to modify it as follows.
This is because there is no case in which the user() function is used in the declaration part.

a varchar := case isnull(USER()) when 0 then 'ok' when 1 then 'nok' end